src/transformState.js

```diff
@@ -62,7 +62,25 @@
  * @param {Object[]} transforms
  */
 function transformState(state, transforms) {
-  // write code here
+  for (const transform of transforms) {
+    switch (transform.operation) {
+      case 'removeProperties':
+        for (const prop of transform.properties) {
```

Comment:
```suggestion
        for (const property of transform.properties) {
```
`prop` is a not very meaningful name